apps/web/src/components/beacon/BeaconPeers.tsx

```diff
@@ -43,32 +29,22 @@ export const BeaconPeers = () => {
     );
   }
 
-  return <PeersDisplay data-testid="beacon-peers" peerInfos={peersWithId} />;
+  return (
+    <VStack
+      alignItems="flex-start"
+      gap="24px"
+      marginTop="24px"
+      data-testid="beacon-peers"
+      divider={<Divider />}
+      spacing="0"
+    >
+      {peers.map(peerInfo => (
+        <PeerRow key={peerInfo.senderId} peerInfo={peerInfo} />
+      ))}
+    </VStack>
+  );
 };
 
-/**
```

Comment:
Just wondering why have deleted this component?

Reply:
just to clean up the code